presto-common/pom.xml

```diff
@@ -91,5 +91,9 @@
             <artifactId>jackson-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Remove this. I think it's no longer needed.

Reply:
Hmm, I thought I removed it but the comment still exists?

Reply:
Done for real now :)